Client/Frontend/Browser/FirefoxTabContentBlocker.swift

```diff
@@ -35,29 +37,30 @@ class FirefoxTabContentBlocker: TabContentBlocker, TabContentScript {
         return "TrackingProtectionStats"
     }
 
-    var isUserEnabled: Bool? {
-        didSet {
-            guard let tab = tab as? Tab else { return }
-            setupForTab()
-            TabEvent.post(.didChangeContentBlocking, for: tab)
-            tab.reload()
-        }
+    override var isEnabledTrackingProtection: Bool {
+        guard let tab = tab as? Tab else { return false }
+        return tab.isPrivate ? self.isEnabledTrackingProtectionInPrivateBrowsing : self.isEnabledTrackingProtectionInNormalBrowsing
```

Comment:
Not super sure about that but it should be safe to remove a separate setting for Private Mode. That is, global settings apply to both normal and private mode. 

Can we please remove those settings for private mode?

Reply:
This is also will be removed in scope of #273